apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/Footer.test.tsx

```
@@ -0,0 +1,48 @@
+'use client';
+
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Footer } from '@/components/Footer';
+
+// Mock Next.js Link
+jest.mock('next/link', () => {
+    return ({ children, href }: { children: React.ReactNode; href: string }) => (
+        <a href={href}>{children}</a>
+    );
```

ESLint

[error] Component definition is missing display name

(react/display-name)